assets/javascript/progressive-reveal.js

```diff
@@ -4,68 +4,66 @@ var each = require('lodash.foreach');
 var groupBy = require('lodash.groupby');
 
 var helpers = require('./helpers'),
-    inputs, groups,
-    toggleAttr = 'data-toggle',
-    hiddenClass = 'js-hidden';
+  inputs, groups,
+  toggleAttr = 'data-toggle',
+  hiddenClass = 'js-hidden';
 
 function inputClicked(e, target) {
-    target = target || helpers.target(e);
-    var shown;
-    each(groups[target.name], function (input) {
-        var id = input.getAttribute('aria-controls');
-        var toggle = document.getElementById(id);
-        if (toggle) {
-            if (input.checked) {
-                input.setAttribute('aria-expanded', 'true');
```

Comment:
I have removed this as per: https://collaboration.homeoffice.gov.uk/jira/browse/RIL-212?focusedCommentId=1258122&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-1258122